docs/modules/ROOT/pages/reference/exoscale-osbapi.adoc

```diff
@@ -0,0 +1,368 @@
+
+The matching `OSBServiceInstance` gets deleted.
+When all `OSBServiceInstance` objects are deleted, an email is sent to customer@vshn.ch for the final closure of the organization.
+
+Also, there is a monitoring check which triggers when no `OSBServiceInstance` is available, but service instances are still there.
+This means the customer didn't delete the service instances.
+
+* TODO: Should we automatically delete service instances after a grace period when there is no `OSBServiceInstance` available?
+
+=== User Synchronization
+
+We don't do https://community.exoscale.com/documentation/vendor/marketplace-managed-services-provision/#user-sync[user synchronization^] from Exoscale to VSHN.
+
+____
+When user sync is disabled, only the information of the user that made the product purchase will be provided. The information will never be updated.
+____
+
+== Instances
+
+=== Provisioning
+
+Instances aren't directly provisioned via the OSB API.
+Instead, an E-Mail is sent to the address in the OSB API provisioning call `parameters.users[0].email` with a well-crafted link to the portal.
+
+See <<Onboarding Flow>> for more details, as it's mostly the same flow.
+
+This portal link encodes:
+
+* The Organization Name (`exo-$UUID`)
+* The `service_id`
+* The `plan_id`
+
+When this portal link is opened, a pre-filled service ordering form is presented in the portal, ready for the user to actually provision the service.
+
+The Exoscale user gets access to all services in the Portal which are available on the Exoscale Origin and can directly provision services from there.
+This allows an Exoscale user to have more than one instance per service per Exoscale organization.
+
+The actual service provisioning happens via the xref:reference/arch-control-plane.adoc[].
+
+E-Mail sending is handled the same way as in https://kb.vshn.ch/appuio-cloud/references/architecture/control-api-invitation.html#_invitation_messages[Invitation^].
+
+=== Plan Change
+
+We don't support plan changes on the Exoscale console, everything is configured on our portal.
+There is only one plan per service, the default plan.
+
+One exception is the "suspension-plan" which is described in the <<Suspension Flow>>.
+
+=== Deprovisioning
+
+See <<Offboarding Flow>> as this is mostly the same flow.
+
+== Billing
+
+The basic flow: We send billing data to Exoscale, Exoscale invoices the end-user, VSHN sends an invoice to Exoscale, Exoscale pays VSHN.
+
+[mermaid]
+....
+flowchart TB
+    exo["Exoscale"]
+    exocust["Exoscale Customer"]
+    vshn["VSHN"]
+
+    exo-- Invoices --> exocust
+    exocust-- Pays -->exo
+
+    vshn-- Invoices --> exo
+    exo-- Pays -->vshn
+....
+
+Exoscale must keep track on our pricing on their end, because we only send usage data and they do the calculation.
+
+TODO
+
+* Send billing data to Exoscale billing API - Exoscale does invoicing to customer - we send invoice to Exoscale
+* One SO on VSHN side for Exoscale, to send invoice to Exoscale
```

Comment:
SO?

Reply:
This part is completely WIP, just ignore it right now

(SO = Sales Order in Odoo)